### PR TITLE
Fix fdb_vlanmac() on corner cases

### PR DESCRIPTION
**Pull request by qiluo-msft**
**- What I did**
Fix fdb_vlanmac() on corner cases

**- How I did it**

**- How to verify it**
Existing unit test

**- Description for the changelog**

### CODE REVIEW

File: src/sonic_ax_impl/mibs/ietf/rfc4363.py [hunk 1]
```diff
@@ -72,6 +79,8 @@ def update_data(self):
             port_id = self.if_bpid_map[bridge_port_id]
 
             vlanmac = self.fdb_vlanmac(fdb)
```
Comment (zhenggen-xu) [inline src/sonic_ax_impl/mibs/ietf/rfc4363.py:81]: should we just "try" these assignments, and capture exceptions instead of making branches in fdb_vlanmac().  And if fdb_vlanmac() is not successful, the rest assignment below should be ignored too.

Comment (qiluo-msft) [reply src/sonic_ax_impl/mibs/ietf/rfc4363.py:81]: Actually linters found the corner cases, and I fixed at the same time to make linting clean.
If you are worried about the CPU cycles, I tried minimize it for the normal cases.
I fixed the loop continue part.

---
In reply to: [564917045](https://github.com/Azure/sonic-snmpagent/pull/193#discussion_r564917045) [](ancestors = 564917045)

**Review by zhenggen-xu — APPROVED** [review]

**Review by SuvarnaMeenakshi — APPROVED** [review]
